### DISCUSSION TITLE
Add Cura-like Smart Skirt

### DISCUSSION THREAD

**Discussion by FormulaNerd3D** [OrcaSlicer/OrcaSlicer#6526] [Ideas]
Recent feature in Cura that I like was the so-called "Smart Skirt". It added a second layer to the skirt line closest to the printed object. It made it easier to remove the skirt, especially when you are printing at smaller layer heights. It would be nice to be able to customize how many layers were added so that it was easier when printing at very low layer heights, e.g., 0.04 and 0.08.

**Comment by vovodroid** [comment]
Is that you are looking for? Available in version 2.2.0

https://github.com/SoftFever/OrcaSlicer/pull/5786

![image](https://github.com/user-attachments/assets/e1d7a516-8399-4ddf-98ff-f90dac748a0c)

Height is adjustable.
